packages/Webkul/Shop/tests/Feature/Checkout/CartTest.php

```diff
@@ -576,6 +576,8 @@
 
 it('should add a downloadable product to the cart', function () {
     // Arrange
+    $this->loginAsCustomer();
```

Comment:
This is part of act not arrange. This line shoule be after act and assert comment.